src/pages/public_registration/PublicRegistrationContent.tsx

```diff
@@ -86,6 +86,13 @@ export const PublicRegistrationContent = ({ registration }: PublicRegistrationCo
 
         <PublicGeneralContent registration={registration} />
 
+        {entityDescription?.alternativeTitles.und && (
+          <Box sx={{ borderTop: '1px solid', py: '1rem' }}>
+            <Typography variant="h3">{t('registration.public_page.alternative_title')}</Typography>
```

Comment:
ser ut som dette skulle vært en h2?
```suggestion
            <Typography variant="h2">{t('registration.public_page.alternative_title')}</Typography>
```

Reply:
eller `variant="h3" component="h2"`?

Reply:
eller det. Spørs om den skal være lik boksen over, eller boksene under da. Jeg tenker kanskje sistnevnte er mest naturlig da, men sikkert greit å forholde seg til skissene der uanZ